### DISCUSSION TITLE
Topic about the tool and the OS virtualization

### DISCUSSION THREAD

**Discussion by paolo-projects** [paolo-projects/auto-unlocker#100] [Q&A]
This section is for discussing about the macOS virtualization topic in general, the tool usage, and any problem you may find. 

---
I hope this topic will be the main place of discussion about any problems you encounter, so we can keep the **Issues** section clean and use it for what it's intended (posting software bugs and feature requests)

**Comment by KINGLIFER** [comment]
Seems to make the guest not able to load after some time using a Mac VM. VMware Workstation for windows.

**Comment by LeFinneus** [comment]
I hit a bit of a snag. Any help would be appreciated! :)

there's a bit of a snag for me. I have Workstation Pro v15.5.7 (yeah, it's a bit old but it's fully licensed and it still works). VMWare Player is no longer available (officially) for download, according to the VMWare website.

In the MasOS Unlocker.exe, I changed the VMWare install locations to the where the folders it is asking for is located on my machine. Everything looks great. I click on Patch and within seconds, I get and error and then opened the log to see what it was.

Here is the log file in full, including the error:

::INFO Killing services and backing up files...
::INFO Stopping services...
::DEBUG Service "VMAuthdService" stopped successfully.
::ERROR Could not kill process "vmware-tray.exe".
::INFO File "C:\Program Files (x86)\VMware\VMware Workstation\vmwarebase.dll" backup done.
::INFO File "C:\Program Files (x86)\VMware\VMware Workstation\x64\vmware-vmx-debug.exe" backup done.
::INFO File "C:\Program Files (x86)\VMware\VMware Workstation\x64\vmware-vmx-stats.exe" backup done.
::INFO File "C:\Program Files (x86)\VMware\VMware Workstation\x64\vmware-vmx.exe" backup done.
::INFO Patching files...
::INFO File: vmware-vmx.exe
::ERROR Couldn't open file C:\Program Files (x86)\VMware\VMware Workstation\x64\vmware-vmx.exe

I don't know where to go from here. I've tried running the x64 vmware-vmx.exe as Administrator and nothing happens. The program will not open. My hunch is that it's because maybe the version of VMWare is too old? There's a steep upgrade price to go to v17.

If you would be so kind as to provide a little help from here, I'd be most appreciative.

Thank you for your time and all your effort. :)
